### PR TITLE
Couple small things

### PR DESCRIPTION
**Pull request by kdzwinel**
I've put runtime filtering in a dropbox because ATM it's taking half of my screen. + other small improvements

compat-table-filtering mov

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,7 +11,7 @@ var runtimes = [
   {
     name: 'RemoteDebug Core',
     protocol: require('./protocols/core/protocol.json'),
-    icon: 'RemoteDebug.png'
+    icon: 'remotedebug.png'
```
Comment (kdzwinel) [inline index.js:14]: ATM it's broken

screen shot 2017-01-25 at 01 34 26

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "standard index.js",
```
Comment (kdzwinel) [inline package.json:7]: no semicolons? 2 spaces? sounds like https://github.com/feross/standard !

File: views/runtimes.ejs [hunk 3]
```diff
@@ -1,18 +1,28 @@
...
-  </div>
+<div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+  <ul class="nav navbar-nav">
+    <li class="dropdown js-runtime-filtering">
+      <a href="#" class="dropdown-toggle"><span class="glyphicon glyphicon-filter" aria-hidden="true"></span> Runtimes <span class="caret"></span></a>
+      <ul class="dropdown-menu">
+        <% for (var i=0; i < runtimes.length - 1; i++) { %>
+        <% var isEnabled = filter.runtimes.includes(runtimes[i].name) %>
+        <li>
+          <a href="#">
```
Comment (kdzwinel) [inline views/runtimes.ejs:9]: bootstrap 😞

File: package.json (hunk 2 above)
Comment (auchenberg) [reply package.json:7]: Yup :)

**Comment by auchenberg** [comment]
LGTM!

**Comment by auchenberg** [comment]
Deployed.

**Pull request merged**